domains/goguard1an.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mg0the0goat00",
+        "email": "s.marley.gooch@redclay.k12.de.us"
+    },
+    "record": {
+        "URL": "https://aboutme-b6sk.onrender.com/"
```

Comment:
I cannot reach your website
"Not Found"

Reply:
changed it to aboutmeeeeeeeeeee.onrender.com im sorry